### ISSUE TITLE
x/pkgsite: importedby group by original / fork

### ISSUE THREAD

**Issue by halfcrazy** [golang/go#63510]
### What is the URL of the page with the issue?
https://pkg.go.dev/github.com/antlr/antlr4/runtime/Go/antlr?tab=importedby

### What is your user agent?
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/117.0.0.0 Safari/537.36

### Screenshot
![image](https://github.com/golang/go/assets/4113173/094b77d5-2abc-4352-acfb-f092ac5687a3)

### What did you do?
open imported by tab

### What did you expect to see?
Repos list group by original repo and fork repo.

### What did you see instead?
It's mixed in one list.

**Comment by thediveo** [comment]
from time to time forks take over because the upstream repo isn't maintained anymore.

**Comment by adonovan** [comment]
cc @jba

**Comment by jba** [comment]
How do you know the forks aren't better than the original? Perhaps one of them _is_ maintained.

**Comment by halfcrazy** [comment]
> How do you know the forks aren't better than the original? Perhaps one of them _is_ maintained.
> 
> 

Yes as upstream repo may no longer maintainerd the fork version is "better". But most case the active  one is the original. I ask for a "group by" not to hide those forks

**Comment by halfcrazy** [comment]
> > How do you know the forks aren't better than the original? Perhaps one of them _is_ maintained.
> 
> Yes as the upstream repo may no longer maintained the fork version is "better". But in most case, the active one is the original. I ask for a "group by" not to hide those forks

Edit:
A special mark may be acceptable. We can distinguish the original and "normal" fork by checking the go.mod's module line.

**Comment by adonovan** [comment]
>  I ask for a "group by" not to hide those forks

Got it; sorry for misunderstanding.
